make.xsh

```diff
@@ -170,10 +173,25 @@ def start_database_stack():
     $POSTGRES_HOST = get_localhost()
     # Deletes any existing pgdata folder and reinitiates it.
     rm -rf infrastructure/containers/postgres/data/pgdata/
-    docker network prune --force
     docker swarm init || echo "docker swarm is already initialized :D"
     docker stack deploy --compose-file docker-compose.yml rootski-database
 
+@makefile.target(tag="run services locally")
+def start_database_stack_lightsail():
+    """
+    Use the "database-backup" service in the "docker-compose.yml" file to backup on a
+    regular inteval specified in the "docker-compose.yml" all of the tables in the
+    databse.
+    """
+    export_dot_env_vars(env_file=DEV_ENV_FILE)
```

Comment:
Should this be `PROD_ENV_FILE` (since it's on the database stack?) But no! I absolutely forgot to say this: the backend API grabs the database credentials from AWS SSM. This is the parameter: 
<img width="861" alt="image" src="https://user-images.githubusercontent.com/32227767/156902046-6d94a04d-e5b5-47f4-bf41-3e19429c514c.png">

Right now... we straight up have our database credentials hard coded in our repository, so the whole world can get into our database. I think we should go ahead and merge this PR when you've decoupled the start and restore targets. But I think right after that, we should actually find a more secure way to get these credentials.

That... is actually going to be awesome :D To do that, you'll actually have to modify the CDK script that you've been running this whole time.

Reply:
Key vault?